addons/payment_ogone/README.md

```diff
@@ -0,0 +1,37 @@
+# Ogone
+
+## Technical details
+
+APIs:
+
+- [Hosted Payment Page](https://support.legacy.worldline-solutions.com/integration-solutions/integrations/hosted-payment-page?com.dotmarketing.htmlpage.language=1&skiprules=true&com.dotmarketing.htmlpage.language=1&skiprules=true)
+- [Direct Link](https://support.legacy.worldline-solutions.com/integration-solutions/integrations/directlink?com.dotmarketing.htmlpage.language=1&skiprules=true&com.dotmarketing.htmlpage.language=1&skiprules=true)
+
+This module relies on a combination of two APIs to implement a payment with redirection flow that
+allows for tokenization. The Hosted Payment Page API is integrated using `payment`'s form-based
```

Comment:
```suggestion
allows tokenization. The Hosted Payment Page API is integrated using `payment`'s form-based
```

Reply:
I want to emphasize that the tokenization is an additional feature.